src/autoqasm/types/types.py

```diff
@@ -152,10 +153,22 @@ def __init__(self, *args, annotations: str | Iterable[str] | None = None, **kwar
         )
         self.name = program.get_program_conversion_context().next_var_name(oqpy.FloatVar)
 
+    def __floordiv__(self, other):
+        raise UnsupportedFeatureError("Integer division is supported by OpenQASM.")
```

Comment:
For these errors, since you don't intend this code to be hit, you should probably raise `NotImplementedError` instead. This will prevent it from being flagged by the code coverage tool (which ignores `NotImplementedError` lines).